conan/api/subapi/config.py

```diff
@@ -76,7 +79,8 @@ def install_pkg(self, ref, lockfile=None, force=False, remotes=None, profile=Non
         # The computation of the "package_id" and the download of the package is done as usual
         # By default we allow all remotes, and build_mode=None, always updating
         conan_api.graph.analyze_binaries(deps_graph, None, remotes, update=update, lockfile=lockfile)
-        conan_api.install.install_binaries(deps_graph=deps_graph, remotes=remotes)
+        with interprocess_write_lock(self.conan_api):
+            conan_api.install.install_binaries(deps_graph=deps_graph, remotes=remotes)
```

Comment:
this is part of the regular Conan packages installation, not affecting the Conan cache configuration yet, why is it locked? In any case, it would make sense to lock everything?

Reply:
Reverted, commit b513786f82e0ebe962c566df1100a2aebe792f3c